### DISCUSSION TITLE
1v1 for free?

### DISCUSSION THREAD

**Discussion by ghost** [cssbattle/forum#207] [Q&A]
I saw the youtubers webdevsimplfied and kevin powell, two pros, 1v1 in css battle, and i wonder, did they host a battle for 199$ each time, or there is a way to play for free?

**Comment by chinchang** [comment]
Oh, thats not really a custom battle. Custom battle specifically is a battle which you organize with your own targets and durations. What you see on Youtube with these awesome people playing is not really a 1v1. Its them just playing a target on their ends and then the videos from both sides combined and edited to give the feel of a live battle between them :)

1v1s are not yet possible in CSSBattle, but....its....gonna....come....soon :) shhhh 🤫

**Comment by ghost** [comment]
oooo thats cool man... Thanks for the answer!!!
